From 6de7a2cc779bc0ccd22013cb1fa1c36d3af3487d Mon Sep 17 00:00:00 2001
From: Adrien Plazas <kekun.plazas@laposte.net>
Date: Sun, 25 Jul 2021 17:09:55 +0200
Subject: [PATCH] WIACompression: Include limits
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This fixes `error: ‘numeric_limits’ is not a member of ‘std’`.
---
 Source/Core/DiscIO/WIACompression.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Source/Core/DiscIO/WIACompression.cpp b/Source/Core/DiscIO/WIACompression.cpp
index 20d19c487707..f39f59ae5b1b 100644
--- a/Source/Core/DiscIO/WIACompression.cpp
+++ b/Source/Core/DiscIO/WIACompression.cpp
@@ -7,6 +7,7 @@
 #include <algorithm>
 #include <cstddef>
 #include <cstring>
+#include <limits>
 #include <memory>
 #include <optional>
 #include <vector>
